shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformChannel.java

```diff
@@ -581,6 +503,149 @@ public SystemChromeStyle(
     }
   }
 
+  /**
+   * A handler of incoming platform channel method calls received from Flutter.
+   * It first determines the platform's API to be called. If it exists, it then
+   * decodes incoming arguments, if needed, into a format that is necessary for the API.
+   */
+  @VisibleForTesting
+  protected class PlatformMethodCallHandler implements MethodChannel.MethodCallHandler {
+    @Override
+    public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result result) {
```

Comment:
I wish the diff were better here. I spot checked it, assuming that nothing in this was changed from the original besides the new method call.